### PR TITLE
fix: Update purple borders on Hierarchical Select

### PR DESCRIPTION
**Pull request by dougmacknz**
# Objective
Updates Hierarchical Select to have gray borders instead of purple (+ a couple of other small design updates).

Also adds a focus ring to this component.

# Motivation and Context
Closes https://github.com/cultureamp/kaizen-design-system/issues/2067

This component doesn't have a design in Figma, so I used the designs for Select, which is very similar.

# Screenshots (if appropriate)
Before
image

After
image

### CODE REVIEW

File: draft-packages/hierarchical-menu/KaizenDraft/HierarchicalMenu/styles.module.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/animation";
 @import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/shadow";
 @import "~@kaizen/deprecated-component-library-helpers/styles/layout";
```
Comment (NatalieLoyzaga) [inline draft-packages/hierarchical-menu/KaizenDraft/HierarchicalMenu/styles.module.scss:6]: Worth removing/replacing the deprecated-component-library-helpers files to use tokens?

Comment (NatalieLoyzaga) [reply draft-packages/hierarchical-menu/KaizenDraft/HierarchicalMenu/styles.module.scss:6]: Saw your comment on slack - Approved this PR for the purpose of updating border colours.

**Review by NatalieLoyzaga — DISMISSED** [review]
LGTM :)

File: draft-packages/hierarchical-menu/KaizenDraft/HierarchicalMenu/styles.module.scss (hunk 1 above)
Comment (dougmacknz) [reply draft-packages/hierarchical-menu/KaizenDraft/HierarchicalMenu/styles.module.scss:6]: Yeah, had a quick look but there's a lot of legacy stuff here, that would be a larger piece of work.

**Review by lloydstubber — APPROVED** [review]

**Pull request merged**